src/player.js

```diff
@@ -147,6 +148,9 @@ class Player {
             screenfull.on('fullscreenchange', this.fullscreenchangeHandler);
         }
 
+        // Add video-specific markup for Google SEO
+        addVideoObjectMarkup(this);
```

Comment:
I should probably add a unit test here after the `videoObjectMetadata` getter has been incorporated.